executor/tests/test_version_cli.py

```
@@ -0,0 +1,104 @@
+
+import pytest
+
+from executor.version import get_version
+
+
+class TestVersionCLI:
+    """Test suite for executor --version CLI flag."""
+
+    def test_version_flag_long(self):
+        """Test that --version flag prints version and exits."""
+        result = subprocess.run(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)